roles/keycloak/tasks/main.yml

```diff
@@ -8,8 +8,12 @@
       name: keycloak
       state: restarted
     become: yes
+  - name: create home/previous folder if not exists
+    file:
+      path: "{{keycloak_home}}/../previous"
+      state: directory
   - name: Move previous installation (folder will be named keycloak-previous-{15-random-alphanumeric-chars} at keycloak_home)
-    command: "mv {{keycloak_home}} {{keycloak_home}}/../keycloak-previous-{{ ansible_date_time.iso8601_basic_short }}"
+    command: "mv {{keycloak_home}} {{keycloak_home}}/../previous/keycloak-previous-{{ ansible_date_time.iso8601_basic_short }}"
```

Comment:
Change this to use the ansible build-in file management tasks, instead of using native mv commands.

Reply:
@NicolasLiampotis I added the "removes" and "creates", however, isn't this something redundant? Isn;t the deletion and creation done "forcibly" under the hood in mv command? Anyway, better safe than sorry ;)